tests/integration/test_integration.py

```diff
@@ -64,12 +65,13 @@ async def test_given_charm_is_built_when_deployed_then_status_is_blocked(
 
 
 async def test_given_charm_is_deployed_when_relate_to_mongo_and_certificates_then_status_is_active(
-    ops_test, build_and_deploy
+    ops_test: OpsTest, build_and_deploy
 ):
-    await ops_test.model.add_relation(
+    assert ops_test.model
```

Comment:
I like this!

Reply:
I wish it could be transitive (not sure if that is the right word) from a fixture. I tried to define 

```
@pytest.fixture
async def ops_test_asserted(ops_test: OpsTest):
    """Assert that the model is ready for testing."""
    assert ops_test.model
    return ops_test
```

to see if it would follow that, but no luck.